cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractAccountApiTest.java

```diff
@@ -422,19 +422,19 @@ private ApiResponse createUser(List<String> subAccountsIds, Boolean enabled) thr
     }
 
     private ApiResponse createUser(List<String> subAccountsIds, Account.Role role) throws Exception {
-        String email = String.format("%s@%s.com", randomLetters(), randomLetters());
+        String email = String.format("sdk+" + SDK_TEST_TAG + randomLetters() + "@cloudinary.com");
         return createUser("TestName", email, role, subAccountsIds);
     }
 
     private ApiResponse createUser(List<String> subAccountsIds, Account.Role role, Map<String, Object> options) throws Exception {
-        String email = String.format("%s@%s.com", randomLetters(), randomLetters());
+        String email = String.format("sdk+" + SDK_TEST_TAG + randomLetters() + "@cloudinary.com");
```

Comment:
Why do you need `String.format()` when there is not `%s` inside?

Reply:
Fixed